content/docs/guides/build/manual-llvm.md

```diff
@@ -0,0 +1,103 @@
+
+```shell
+choco install --confirm git golang mingw make cmake ninja python
+```
+**Windows** users should also use *Git Bash* (installed above) to run all the build commands like `make`. The TinyGo build system expects a Unix-like environment that is not normally provided by Windows but is included already in *Git Bash*.
+
+Choco doesn't seem to add CMake automatically to the `$PATH` variable. You can do this manually if needed, in Git bash:
+
+```shell
+export PATH="$PATH:/c/Program Files/CMake/bin"
+```
+
+## Building LLVM
+***The following instructions are common to all operating systems.***
+
+Background: *[LLVM](https://llvm.org/) is used by TinyGo to create smaller programs than can be compiled with normal Go distribution. It is commonly referred to as a *compiler infrastructure* since it provides several tools to build, link and optimize programs which adhere to its framework.*
+
+We build LLVM from inside the TinyGo repository we cloned in the [previous step](../). The following command when run inside the TinyGo repo will take care of first downloading the LLVM source code to later build it in the next step. It places the source code in `llvm-project/` and the build output in `llvm-build/`. It only needs to be done once until the next LLVM release (every half year).
+
+Note that the build step may take some time- feel free to grab a drink meanwhile. Warnings emitted through the compilation in this part are normal as of LLVM 17.
+
+```shell
+make llvm-source llvm-build
+```
+
+When building on Windows, add CCACHE=OFF.
+
+```shell
+make llvm-source llvm-build CCACHE=OFF
+```
+
+
+## Building TinyGo
+
+Once this is finished, you can build TinyGo against this manually built LLVM:
+
+```shell
+make
+```
+
+This results in a `tinygo` binary in the `build` directory:
+
+```shell
+$ ./build/tinygo version
+tinygo version 0.31.0-dev-d4189fec linux/amd64 (using go version go1.21.4 and LLVM version 16.0.1)
+```
+
+You have successfully built TinyGo from source. Congratulations! What's left now is to complete the [additional requirements](../additional-requirements)
+
+#### Adding tinygo to your path
+
+**Linux** users may choose to run TinyGo from any directory you may want to move the built binary to a location on your path
+or add the `./build` directory to your path. The following shell command moves the TinyGo binary to `/user/bin` so that any user can run TinyGo. You may need root privileges to complete this step.
+```shell
+mv ./build/tinygo /usr/bin/
+```
+If you ran the above command, **uninstalling** TinyGo is as easy as running `rm /usr/bin/tinygo`.
+
+If you'd prefer a user-based install you can move TinyGo to your `$HOME/go/bin` directory, where the Go compiler installs binaries by default. The directory should be added to your PATH if it hasn't already been added for this to work.
```

Comment:
Perhaps just saying "copy to $GOBIN" would be better? That will work for the majority of people with none of the issues of moving it to `/usr/bin`. Also, it means it's not really necessary to uninstall TinyGo first.

Reply:
Copying to $GOBIN has been working for me for quite a while.

Reply:
Thing is $GOBIN is not defined by default, so most users who are new to Go would have issues with that step.

Reply:
That is true, but it is a one line export that we could include in the instructions.